app/services/invoices/refresh_draft_service.rb

```diff
@@ -60,6 +55,7 @@ def call
           recurring:,
           context:
         )
+        return calculate_result if tax_error?(calculate_result.error)
```

Comment:
shouldn't we return error result after iterating over credit notes?

Reply:
Because you had concerns with credit notes, I thought it'll be better to return before 🙈 
can move it after since anyway we'll be implementing logic on credit notes 👍

Reply:
for now both works, but please add note if it will stay as it is

Reply:
updated